status-im/status-react
# Bug/improved wallet styles

### Summary:

- improved wallet name style
- do not use default amount for wallet detail
- fixed broken button style

### Steps to test:
- Open Status
- Navigate wallet tab
- Verify long wallet name are shorten in the middle
- Verify wallet details amount is correct
- Verify wallet button are on a single line

status: ready
